homework_03/12_Martin_Vrachev/test_circles.py

```diff
@@ -0,0 +1,81 @@
+from circles import *
+import pytest
+
+def test_create_point():
+    p1 = Point(5.4, 6.5)
+    assert isinstance(p1, Point)
+
+def test_create_invalid_point():
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        p2 = Point('koko', 5.7)
+
+    assert pytest_wrapped_e.type == SystemExit
+
+def test_create_circle():
+    p = Point(5.5, 4.5)
+    c = Circle(p, 10.0)
+    assert isinstance(c, Circle)
+
+def test_create_invalid_circle():
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
```

Comment:
It is not necessary to write this on every test. You could add it on the top of the class(which you don't have) as global variable.

Reply:
agreed great tip